dm/docs/RFCS/20220124_enhance_task_manageability.md

```diff
@@ -0,0 +1,127 @@
+# Proposal: Enhance task manageability
+
+- Author(s): [ehco](https://github.com/Ehco1996)
+- Last updated: 2022-01-22
+
+## Background
+
+The main purpose of this change is to address the root cause of problems like [#3771](https://github.com/pingcap/tiflow/issues/3771), which are caused by the fact that DM task itself does not distinguish between dynamic configuration and static resources, making it impossible for users to intuitively manage their tasks. For this reasons, we will attempt to redesign the state machine of DM task and optimize the dmctl interaction interface to provide a better user experience.
+
+### Current State machine of task
+
+![dm old task lifecycle](../media/dm-old-task-lifecycle.png)
+
+### New State machine of task
+
+![dm new task lifecycle](../media/dm-new-task-lifecycle.png)
+
+This is mainly achieved by adding a new `create task` action to create a new task in a stopped state, rather than directly creating and starting a task with `start-task` .
+
+## Goals
```

Comment:
please use one grammatical tense in three items below.

Reply:
you mean `start-task` and `create task`?

Reply:
do/doing/did

Reply:
ping @Ehco1996

Reply:
emm I don't know how to edit this sentence, I asked my English major, she said the tense of this sentence is present perfect tense

Can you suggest any changes?